src/main/scala/codacy/plugins/test/TestFilesParser.scala

```diff
@@ -79,10 +79,18 @@ class TestFilesParser(filesDir: File) {
 
               comment.trim match {
                 case IssueWithLineRegex(value) =>
-                  for {
-                    IssueWithLine(severityStr, line, patternId) <- Json.parse(value).asOpt[IssueWithLine]
-                    severity <- ResultLevel.values.find(_.toString.startsWith(severityStr))
-                  } yield TestFileResult(patternId, line, severity)
+                  Try {
+                    for {
+                      IssueWithLine(severityStr, line, patternId) <- Json.parse(value).asOpt[IssueWithLine]
+                      severity <- ResultLevel.values.find(_.toString.startsWith(severityStr))
+                    } yield TestFileResult(patternId, line, severity)
+                  } match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-plugins-test/file/1364015748/issues/source?bid=2836238&fileBranchId=2974430#l87)